Dockerfile

```diff
@@ -1,5 +1,20 @@
-FROM sonarqube:10.5.0-community
+ARG MAVEN_BUILDER=3-openjdk-17-slim
+#ARG SONARQUBE_VERSION=24.12.0.100206-community
```

Comment:
I tested with version 24.12 and it is ok : we can move to it

Reply:
update done on branch